<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwnd</class>
 <widget class="QMainWindow" name="mwnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="RenderPanel" name="renderpanel" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Компоненты</string>
    </property>
    <addaction name="actionPoint"/>
    <addaction name="actionCircle"/>
    <addaction name="actionClearAll"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="text">
    <string>Точка</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="text">
    <string>Ломаная</string>
   </property>
  </action>
  <action name="actionPoly">
   <property name="text">
    <string>Полигон</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Окружность</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>Очистить всё</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderPanel</class>
   <extends>QWidget</extends>
   <header>renderpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
